test/controllers/hostgroups_controller_test.rb

```diff
@@ -264,4 +267,19 @@ def setup_user(operation, type = 'hostgroups')
       end
     end
   end
+
+  context 'with pagelets' do
+    setup do
+      @controller.prepend_view_path File.expand_path('../../static_fixtures', __FILE__)
+      Pagelets::Manager.add_pagelet('hostgroups/edit', :main_tabs,
```

Comment:
Since the usage would be to both edit and new screens, I think we should test the more realistic behavior - registering to `hostgroup/_form` and consuming from both new and edit screens.
[Here](https://github.com/theforeman/foreman/blob/develop/config/initializers/puppet.rb#L2) is an example of such use for hosts form.